WebContent/js/containers/ContentViewer.js

```diff
@@ -92,7 +100,22 @@ export class ContentViewer extends React.Component {
         dispatch(removeContent(removeIndex));
         // Do we need to change the selectedContent
         if (removeIndex <= selectedContent && selectedContent >= 1) {
-            dispatch(changeSelectedContent(selectedContent - 1));
+            const newTabIndex = selectedContent - 1;
+            dispatch(changeSelectedContent(newTabIndex));
+        }
+    }
+
+    handleKeyDown(e, index) {
+        if (e.key === 'Enter') { this.handleSelectedTabChange(index); }
+        // if (e.altKey && e.keyCode === 87) { this.handleCloseTab(index); }
```

Comment:
We shouldn't include commented out code which is unused, it will go stale

Reply:
i think we need global key handler for closing tab, so no matter where the focus is, so decided to remove it, left it as reminder, removed now